<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #B8E0FF;
}

QWidget {
	background-color: #B8E0FF;
}

QPushButton {
    border-radius: 0px;
    border: 2px solid #0078d4;
    background-color: #0078d4;
    color: #ffffff;
    padding: 3px 14px;
}

QPushButton:hover {
    background-color: #ffffff;
	color: #0078d4;
}

QPushButton:pressed {
    background-color: #ffffff;
	color: #ffffff;
}

QLineEdit {
	border-radius: 10px;
	padding: 10px 14px;
	margin: 2px;
	background-color: #ffffff;
}

QCheckBox::indicator {
	border-radius: 5px;
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff;
    border: 2px solid #0078d4;
}

QCheckBox::indicator:checked {
    background-color: #0078d4;
    border: 2px solid #0078d4;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="pathEdit">
        <property name="placeholderText">
         <string>No File Selected...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="chooseBttn">
        <property name="text">
         <string>Select File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="imageframe">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>280</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="text">
         <string>No File Selected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="backBttn">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isImageCB">
        <property name="text">
         <string>This Document is Mostly Imagery</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="processBttn">
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="logsBttn">
        <property name="text">
         <string>View Logs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createBttn">
        <property name="text">
         <string>Create Account</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>chooseBttn</tabstop>
  <tabstop>backBttn</tabstop>
  <tabstop>isImageCB</tabstop>
  <tabstop>processBttn</tabstop>
  <tabstop>logsBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
